packages/api/endpoints/granules.js

```diff
@@ -140,37 +142,48 @@ async function del(req, res) {
   log.info(`granules.del ${granuleId}`);
 
   const granuleModelClient = new models.Granule();
+  const granulePgModel = new GranulePgModel();
+  const filePgModel = new FilePgModel();
 
-  let granule;
+  const knex = await getKnexClient({ env: process.env });
+
+  let dynamoGranule;
+  let pgGranule;
   try {
-    granule = await granuleModelClient.getRecord({ granuleId });
+    dynamoGranule = await granuleModelClient.getRecord({ granuleId });
+    pgGranule = await granulePgModel.get(knex, { granule_id: granuleId });
   } catch (error) {
     if (error instanceof RecordDoesNotExist) {
       return res.boom.notFound(error);
     }
     throw error;
   }
 
-  if (granule.detail) {
-    return res.boom.badRequest(granule);
+  // DEPRECATED this can be removed
```

Comment:
Remove?

Reply:
Yeah, so I asked Mark about this here:

https://github.com/nasa/cumulus/pull/2057#discussion_r568945504

And it sounds like this can be removed. He had a good point that it doesn't totally need to be done in the scope of this PR so my plan was to finish up here then go back, remove this, and see if anything breaks.

Reply:
> my plan was to finish up here then go back, remove this, and see if anything breaks.

Not clear on if you mean in a separate PR - if so, feel free to resolve this comment chain, else this is still outstanding.

Reply:
Ah yeah, I think either way this will be in a separate PR. Resolving this for now.